daprdocs/content/en/developing-applications/building-blocks/observability/app-health.md

```diff
@@ -2,41 +2,45 @@
+- Could be too busy to accept new work;
+- Could have crashed; or 
+- Could be in a deadlock state. 
 
-When app health checks are enabled, the Dapr *runtime* (sidecar) periodically polls your application via HTTP or gRPC calls.
+Sometimes the condition can be transitory, for example:
+- If the app is just busy and will resume accepting new work eventually
+- If the application is being restarted for whatever reason and is in its initialization phase
 
-When it detects a failure in the app's health, Dapr stops accepting new work on behalf of the application by:
+App health checks are disabled by default. Once you enable app health checks, the Dapr runtime (sidecar) periodically polls your application via HTTP or gRPC calls. When it detects a failure in the app's health, Dapr stops accepting new work on behalf of the application by:
 
 - Unsubscribing from all pub/sub subscriptions
 - Stopping all input bindings
 - Short-circuiting all service-invocation requests, which terminate in the Dapr runtime and are not forwarded to the application
 
 These changes are meant to be temporary, and Dapr resumes normal operations once it detects that the application is responsive again.
 
-App health checks are disabled by default.
-
 <img src="/images/observability-app-health.webp" width="800" alt="Diagram showing the app health feature. Running Dapr with app health enabled causes Dapr to periodically probe the app for its health.">
 
-### App health checks vs platform-level health checks
+## App health checks vs platform-level health checks
 
 App health checks in Dapr are meant to be complementary to, and not replace, any platform-level health checks, like [liveness probes](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/) when running on Kubernetes.
 
 Platform-level health checks (or liveness probes) generally ensure that the application is running, and cause the platform to restart the application in case of failures.  
+
 Unlike platform-level health checks, Dapr's app health checks focus on pausing work to an application that is currently unable to accept it, but is expected to be able to resume accepting work *eventually*. Goals include:
 
 - Not bringing more load to an application that is already overloaded.
 - Do the "polite" thing by not taking messages from queues, bindings, or pub/sub brokers when Dapr knows the application won't be able to process them.
 
 In this regard, Dapr's app health checks are "softer", waiting for an application to be able to process work, rather than terminating the running process in a "hard" way.
 
-> Note that for Kubernetes, a failing App Health check won't remove a pod from service discovery: this remains the responsibility of the Kubernetes liveness probe, _not_ Dapr.
+> **Note:** For Kubernetes, a failing App Health check won't remove a pod from service discovery: this remains the responsibility of the Kubernetes liveness probe, _not_ Dapr.
```

Comment:
Shouldn't this be in the "Standard" note format with the side bar rather than as bold?
{{% alert title="Note" color="primary" %}}
like this
{{% /alert %}}

Reply:
yep, not sure why I glossed over that - updating